syft/pkg/cataloger/deb/parse_dpkg_status.go

```diff
@@ -133,7 +133,8 @@ func extractAllFields(reader *bufio.Reader) (map[string]interface{}, error) {
 			var val interface{}
 			key, val, err = handleNewKeyValue(line)
 			if err != nil {
-				return nil, err
+				log.Warnf("parsing key from line: %s err: %v", line, err)
```

Comment:
Same comment from earlier, as an error with chaining there is enough context to glean where the message is coming from. As a log warning there isn't enough context here to tell it's a dpkg status failure when appearing in the terminal.

Reply:
The log line should have more context info now